libs/design/src/atoms/form/quantity-field/quantity-field.component.ts

```diff
@@ -55,7 +55,15 @@ export class DaffQuantityFieldComponent implements ControlValueAccessor, DaffFor
    */
   @Input() selectMax = 10;
 
-  focused = false;
+  get focused() {
+    if (this.input || this.select) {
+      return this.input?.focused || this.select?.focused;
+    }
+
+    // If we don't have a control, quantity field should not be focused.
+    return false;
```

Comment:
```ts
return this.input?.focused || this.select?.focused
```
should be an equivalent implementation. Additionally there should be tests for this behavior.